src/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -6,18 +6,19 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.1-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.1-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.3-Debug|AnyCPU'" />
```

Comment:
Why did we have to bump up the netandard version? What API was needed?

Reply:
I'm going to revert.

Reply:
As @ahsonkhan suggests this is wrong. These configurations have to match with what we have on the Configurations.props sitting right next to the project, and we can't drop support of netstandard1.1